### PR TITLE
NMS-16131: Added ed25519 algorithm

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://opennms.atlassian.net/browse/NMS-16131

### CODE REVIEW

**Comment by RangerRick** [comment]
Hrm, looks like something going weird with jsch in the smoke tests? Could this be related to your changes? At first glance doesn't seem like it should be, but...

**Review by Bonrob2 — COMMENTED** [review]
Got auto tagged for review, but I don't see any docs, so deferring to others.

File: container/karaf/src/main/filtered-resources/etc/custom.properties [hunk 1]
```diff
@@ -39,6 +39,7 @@ org.osgi.framework.system.packages.extra=org.apache.karaf.branding,\
         sun.net.dns,\
         sun.net.spi.nameservice,\
         sun.reflect,\
+        sun.security.x509,\
```
Comment (RangerRick) [inline container/karaf/src/main/filtered-resources/etc/custom.properties:42]: You will probably have to do this in the minion and sentinel versions of `custom.properties` as well. (`features/container/sentinel/src/main/filtered-resources/etc/custom.properties` and `features/container/minion/src/main/filtered-resources/etc/custom.properties`)

Comment (RangerRick) [reply container/karaf/src/main/filtered-resources/etc/custom.properties:42]: Someday we should probably rework this stuff to be a template so we don't have to keep doing all the same things in 3 different places...

**Comment by christianpape** [comment]
Yay! CircleCI is happy!

**Review by RangerRick — APPROVED** [review]

**Comment by christianpape** [comment]
Rebased to foundation-2023.

File: container/karaf/src/main/filtered-resources/etc/custom.properties (hunk 1 above)
Comment (christianpape) [reply container/karaf/src/main/filtered-resources/etc/custom.properties:42]: done

**Review by RangerRick — APPROVED** [review]
lookin' good!

**Pull request merged**